### DISCUSSION TITLE
Intermittent crash when starting nightly

### DISCUSSION THREAD

**Discussion by dsl101** [FreeRDP/FreeRDP#10061] [Q&A]
Not posting this as a bug report, as I don't have any details. But about 1 in 5 launches result in this kind of thing:

```
$ xfreerdp3 /v:10.0.0.121 /size:1280x800 /sound /mic:format:0 /log-level:debug | tee xfree-mic-format-0.log
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer:DEADLYSIGNAL
.
.
.
```

and so on until I kill the process. Simply rerunning the command line _usually_ results in a successful connection. Anything I can do to identify what's going on?

Other details:

OS: Antix with 6.8.4-2-liquorix-amd64 #1 ZEN SMP PREEMPT liquorix 6.8-2.1~bookworm (2024-04-08) x86_64 GNU/Linux
xfreerdp3: 3.4.1-dev0 (d864393a8)

**Comment by akallabeth** [comment]
1. you might want to install `freerdp-nightly-dbg` for debug symbols
2. are you sure your os is compatible with `debian bookworm` packages without recompile?

**Comment by dsl101** [comment]
OK, will try installing that. Antix is Debian 12.5 based, so I was presuming so...
```
$ cat /etc/antix-version
antiX-23_x64-core Arditi del Popolo 27 August 2023
$ cat /etc/debian_version
12.5
```

Edit: Other than installing `freerdp-nightly-dbg`, is there anything I need to do? Just get the same repeated output on a bad run.

**Reply by akallabeth** [reply]
well, update, your nightly is 2 weeks old.
but strange, if there is a deadly signal it should crash with some more output than you have.

**Comment by dsl101** [comment]
I'm running KDE which shows updates via Discover daily, and I thought it was keeping it up to date. But even via `apt` it's showing as the latest version. Some other info:
```
# dpkg -s freerdp-nightly
Package: freerdp-nightly
Status: install ok installed
Priority: optional
Section: x11
Installed-Size: 18600
Maintainer: FreeRDP <team@freerdp.com>
Architecture: amd64
Version: 3.0.0+0~20240327024821.1506~1.gbpd86439
.
.
.

# apt-cache showpkg freerdp-nightly
Package: freerdp-nightly
Versions:
3.0.0+0~20240327024821.1506~1.gbpd86439 (/var/lib/apt/lists/pub.freerdp.com_repositories_deb_bookworm_dists_freerdp-nightly_main_binary-amd64_Packages) (/var/lib/dpkg/status)
```

Has the nightly package stopped updating somewhere maybe?

On the error, I had to change Konsole settings for unlimited history, as even hitting CTRL+C immediately after running the command I had more than 1000 lines of `AddressSanitizer:DEADLYSIGNAL` but I managed to get back to the command itself, and there was no other output before all those.

Is there anything else you can think of I can try?

**Comment by dsl101** [comment]
So, today there was an advertised update to nightly in KDE Discover. I now see version `3.4.1-dev0 (f6a18d142)`. Is that current?

Still seeing the occasional failure on startup though.

**Reply by akallabeth** [reply]
@dsl101 ok, then please add a log of such a run. (please do a `2>&1 |tee file.txt` to catch all, `stdout` and `stderr`)
